package.json

```diff
@@ -617,7 +617,7 @@
 		"watch": "tsc -p -watch ./",
 		"test": "npm run compile && node ./out/test/runTest.js",
 		"cc": "npm run compile && node ./out/test/runTest.js --codecoverage",
-		"e2etest": "npm run compile && extest setup-and-run -c 1.50.0 ./out/test/e2e/*.js -m ./out/test/e2e/config.js"
+		"e2etest": "npm run compile && extest setup-and-run -c 1.50.0 ./out/test/e2e/*.js -m ./out/test/e2e/config.js -u"
```

Comment:
just curious, what is "-u" for?

Reply:
It's to uninstall the extension after e2e tests are done,[ as described here](https://github.com/redhat-developer/vscode-extension-tester/wiki/Test-Setup).

Reply:
On the development environment, we don't want it installed, so this -u flag is helpful to clean it up.